### ISSUE TITLE
defaultDisplayOption not working

### ISSUE THREAD

**Issue by RaphaelHofer** [sulu/sulu-standard#725]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.2.3
| Browser Version | Chrome 51.0.2704.106

#### Actual Behavior

The defaultDisplayOption is visible at the media_selection content type, but the view returns "top".

#### Expected Behavior

returning the right displayOption

#### Steps to Reproduce

Define a block with a media_selection and disable all displayOptions except left and right. Set the defaultDisplayOption to left or right.

#### Possible Solutions

Click one of the displayOptions before saving

**Comment by wachterjohannes** [comment]
this option will be passed to the javascript component but wont be used there. we have to change the way we pass this option to the javascript component which uses `this.options.dataDefaults` to initialize data.

**Comment by wachterjohannes** [comment]
@marcelmoosbrugger i think this would be a quite easy fix for 1.3 /cc @danrot

**Comment by marcelmoosbrugger** [comment]
I'd suggest that we wait on fixing this, as Bernd planned a lot of new stuff for the media bundle.

**Comment by danrot** [comment]
@marcelmoosbrugger I don't think that the changes Bernd has planned will require us to hold back this change.
